### PR TITLE
new keyboard rb87

### PR DESCRIPTION
**Pull request by ryanbaekr**
## Description

Added new keyboard: ryanbaekr rb87

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ryanbaekr/rb87/keymaps/default/readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# The default keymap for rb87
```
Comment (waffle87) [inline keyboards/ryanbaekr/rb87/keymaps/default/readme.md:1]: This file should be removed.

File: keyboards/ryanbaekr/rb87/readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+A diy stacked acrylic 80% keyboard. 
+
+* Keyboard Maintainer: [ryanbaekr](https://github.com/ryanbaekr)
+* Hardware Supported: rb87 with Elite-C
+* Hardware Availability: 2022/12/30
+
+Make example for this keyboard (after setting up your build environment):
+
+    make ryanbaekr/rb87:default
```
Comment (waffle87) [inline keyboards/ryanbaekr/rb87/readme.md:11]:
```suggestion
    make ryanbaekr/rb87:default
    
Flashing example for this keyboard:

    make ryanbaekr/rb87:default:flash
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/ryanbaekr/rb87/info.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+    "keyboard_name": "rb87",
+    "manufacturer": "ryanbaekr",
+    "url": "",
+    "maintainer": "ryanbaekr",
```
Comment (waffle87) [inline keyboards/ryanbaekr/rb87/info.json:5]:
```suggestion
    "maintainer": "ryanbaekr",
    "processor": "atmega32u4",
    "bootloader": "caterina",
```

**Review by drashna — APPROVED** [review]

File: keyboards/ryanbaekr/rb87/config.h [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 17
+
```
Comment (fauxpark) [inline keyboards/ryanbaekr/rb87/config.h:25]:
```suggestion
```

File: keyboards/ryanbaekr/rb87/rules.mk [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = caterina
+
```
Comment (fauxpark) [inline keyboards/ryanbaekr/rb87/rules.mk:6]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**